lib/Graph/Node.cpp

```diff
@@ -80,11 +80,6 @@ void Node::setPredicate(const NodeValue &P) { predicate_ = P; }
 bool Node::hasPredicate() const { return predicate_.getNode(); }
 
 TypeRef Node::getType(unsigned idx) const {
-  if (idx == (unsigned)-1) {
-    assert(numRes_ == 1 && "Did not specify the result number for a node "
-                           "with multiple results.");
-    return types_[0];
-  }
```

Comment:
That's the code I was talking about :).
Thanks!

Out-of-curiosity, do you know why the coverage bot, reports this as covered? (That was what I was checking when I asked you about our coverage checking.)

Reply:
Not sure, will look into this.